source/mapping-rules/fields/field-customizations.txt

```diff
@@ -0,0 +1,102 @@
+.. _rm-field-customizations:
+
+======
+Fields
+======
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can customize fields when adding or editing your mapping rules. 
+Relational Migrator allows you to define custom fields using 
+:ref:`rm-calculated-fields`, define custom IDs using 
+:ref:`rm-id-field-customizations`, or convert your source 
+relational column data types into :ref:`a supported BSON data type<rm-field-supported-types>`.
+
+Use Cases
+---------
+
+You can use field customization to:
+
+- Include or exclude fields in your mapping rules.
+- Change field names.
+- Rename fields to include a dotted path, which results in 
+  that field embedded in a parent document. For details, see 
+  :ref:`embedding fields<rm-embedding-individual-fields>`.
+- Customize ``_id`` fields using JavaScript expressions.
+- Customize a field using JavaScript expressions.
+- Configure null handling behavior.
+
+Behavior
+--------
+
+The fields section contains a list of fields derived from columns 
+in the relational table. Mapping rules include each column by default. 
+You can exclude a field from your sync job by unchecking the box. 
```

Comment:
[suggestion] You might consider a global search for "by" in order to edit these "by ...-ing" constructions out. 

"To exclude a field from your sync job, uncheck the box."